source/LibUISharp/src/LibUISharp/TabControl.cs

```diff
@@ -13,7 +13,7 @@ public class TabControl : ContainerControl<TabControl, TabPageCollection>
 
     public class TabPage : Control
     {
-        protected Control child;
+        protected Control childField;
```

Comment:
What was this change for? (not against changing the name or anything, just curious)

Reply:
CLSCompliance because some .net languages aren't case-sensitive. And apparently preceding it with an underscore wasn't good enough.

Reply:
Any way this can be renamed to `@child`? I don't know if that will cause any more warnings or not.